src/main/java/net/minecraftforge/common/capabilities/CapabilityDispatcher.java

```diff
@@ -152,4 +162,43 @@ public void invalidate()
     {
         this.listeners.forEach(Runnable::run);
     }
-}
+
+    @Override
+    public void encode(PacketBuffer out, boolean writeAll)
+    {
+        for (Map.Entry<String, INetworkCapability> entry : networkCapabilities.entrySet())
+        {
+            if (!writeAll && !entry.getValue().requiresSync())
+                continue;
+            out.writeUtf(entry.getKey(), 0x100);
```

Comment:
shouldn't use an constant for the 0x100 ?

Reply:
It's the constant used by vanilla in `PacketBuffer::writeUtf(String)`, `PacketBuffer::readResourceLocation` and `PacketBuffer::readUtf`. I'm not using the 'lengthless' write methods as `PacketBuffer::readUtf` is stripped from the server. I'm not sure what I'd call the constant if I pulled it out into a variable because I don't know what its significance is.

Reply:
I meant declare a static final for it.

Reply:
Yeah I know, not sure what I'd call it though.